pinax/blog/models.py

```diff
@@ -189,14 +193,22 @@ def save(self, **kwargs):
         super(Post, self).save(**kwargs)
 
     @property
+    def scoping_url_kwargs(self):
```

Comment:
@brosner all fixed up but i still find this property useful on the `Post` object.